File: 백준/Bronze/4153. 직각삼각형/직각삼각형.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        int[] sides = new int[3];
        while (true) {
            sides[0] = sc.nextInt();
            sides[1] = sc.nextInt();
            sides[2] = sc.nextInt();
            if (sides[0] == 0 || sides[1] == 0 || sides[2] == 0) {
                break;
            } else {
                Arrays.sort(sides);
                if (sides[2] * sides[2] == sides[1] * sides[1] + sides[0] * sides[0]) {
                    System.out.println("right");
                } else {
                    System.out.println("wrong");
                }
            }
        }
        sc.close();

    }
}

File: 백준/Bronze/4153. 직각삼각형/직각삼각형.py
while True:
    sides = list(map(int, input().split()))
    if sides[0] == 0 or sides[1] == 0 or sides[2] == 0:
        break
    else:
        sides.sort()
        if sides[2]**2 == sides[1]**2 + sides[0]**2:
            print("right")
        else:
            print("wrong")